### ISSUE TITLE
[Asus TF300T] Recovery doesn't work

### ISSUE THREAD

**Issue by simonbcn** [TeamWin/Team-Win-Recovery-Project#133]
I have updated my Asus TF300T to the last official Android version of Asus: 4.2.1
After this process I have to install again the recovery. I have tried two versions: 2.3.3.0 and 2.4.4.0 and both fails in the same: it asks me a password when it boots recovery and it's unabled to mount any partition.

**Comment by simonbcn** [comment]
I have tested the old version v2.2.1.3-tf300t and it doesn't start.

**Comment by Dees-Troy** [comment]
See last comment in:
https://github.com/TeamWin/Team-Win-Recovery-Project/issues/138

**Comment by simonbcn** [comment]
ok, thank you too much. Now it works. :+1:

**Comment by julianikos** [comment]
I looked at #138, installed linked version of twrp 2.4.4.0-tf300t-4.2.blob. I'm prompted for pasword, only option I know is hit cancel. Unable to see SD card still. Tried almost every version of twrp. This is after tf300t updated to 4.2.1 US10.6.1.8. What did I miss that simonbcn didn't ? Thank you, Julian

**Comment by simonbcn** [comment]
My installed firmware versión is: `V10.6.1.8 Only for WW SKU (Android 4.2)`.
I previously had installed several twrp versions to test without success.

**Comment by mattrsch** [comment]
I am having the same issue as julianikos.  The linked version does not work either

**Comment by julianikos** [comment]
I have an update on my SD issue with twrp 2.4.4.0 and tf300t 4.2.1 US10.6.1.8. I have bought a new SD card 32GB Sandisk Ultra class 10 and it mounted just fine in TWRP. My Android phone has the same make just 16GB and that SD card works as well in twrp. I tried a 16GB class 4 sandisk from my GF's Android tablet and that card works also. So at this stage I think something is up with the 8GB card that I had in it, it did work prior the update, but I tried formatting, partition it, reformat same as the other working SD's and it still won't mount in twrp. I'm okay assuming that the 8GB SD is bad at this point do to the fact that  all other SD cards work (class4, class10, ultra etc.) 
I can only recommend to try a different SD card at this point. I'm not an expert but perhaps during the update something happened to the card etc. I know SD's can go flaky ! So the linked version of TWRP does work for me I have to say, just not on one out of three SD cards I have tried. 3 SD's work 1 does not = SD problem !!??
